cmd/import.go

```diff
@@ -15,6 +19,11 @@ var importCmd = &cobra.Command{
 	Run:   runImport,
 }
 
+var allowedImports = map[string](string){
+	"CHANNELS": "sql_statements.sql",
```

Comment:
I'm not sure about this. We have the goal of import all or nothing, even if we have different entities. Maybe we should write all data to the same export file, and just call one.